mcd/mcd-seth/mcd-seth.md

```diff
@@ -125,192 +167,273 @@ In order to better understand the MCD contracts, the following provides a brief
+First we are going to make a transaction to the WBTC adapter to actually take 5 of our tokens with the join contract function.
+The contract function looks like the following: `join(address urn, uint256 amt)`.
 
-- The first parameter is the `urn`, our vault address.
+- The first parameter is the `urn`, our vault address
 - The second parameter is the token amount in `wad`.
 
-For the sake of readability, we up the `wadC` parameter representing the amount of collateral:
+For the sake of readability, we set the `amt` parameter representing the amount of collateral:
+
+```bash
+export amt=$(seth --from-fix $WBTC_DECIMALS 5)
+```
 
-`export wadC=$(seth --to-uint256 $(seth --to-wei 150 eth))`
+Then use the following command to use the join function, thus taking 5 WBTC from our account and sending to `urn` address.
 
-Then use the following command to use the join function, thus taking 150 BAT from our account and sending to `urn` address.
+```bash
+seth send $MCD_JOIN_WBTC_A 'join(address, uint256)' $urn $amt
+```
 
-`seth send $MCD_JOIN_BAT_A "join(address, uint)" $urn $wadC`
+**ℹ️ NOTICE:** From this point on, the [join-5](https://goerli.etherscan.io/address/0x3cbE712a12e651eEAF430472c0C1BF1a2a18939D#code) adapter already took care of the fact that WBTC has only 8 decimals, so we can proceed with `wad` normally.
 
-We can check the results with the contract function: `gem(bytes32 ilk,address urn)(uint256)` with.
+We can check the results with the contract function: `gem(bytes32 ilk, address urn)(uint256)` with:
 
-`seth --from-wei $(seth call $MCD_VAT 'gem(bytes32,address)(uint256)' $ilk $urn) eth`
+```bash
+seth call $MCD_VAT 'gem(bytes32, address)(uint256)' $ilk $urn | seth --from-wei
+```
 
 The output should look like this:
 
-`150.000000000000000000`
+`5.000000000000000000`
 
-An optional, but recommended step is to invoke `jug.drip(ilk)` to make we are not paying undue stability fees. For more detail, please read the guide [Intro to the Rate mechanism](https://github.com/makerdao/developerguides/blob/master/mcd/intro-rate-mechanism/intro-rate-mechanism.md)
+An optional, but recommended step is to invoke `jug.drip(ilk)` to make we are not paying undue stability fees. For more detail, please read the guide [Intro to the Rate mechanism](../intro-rate-mechanism/intro-rate-mechanism.md).
 
-`seth send $MCD_JUG 'drip(bytes32)' $ilk`
+```bash
+seth send $MCD_JUG 'drip(bytes32)' $ilk
+```
 
 The next step is adding the collateral into an urn. This is done through the `CDP Manager` contract.
-The function is called `frob(uint256,uint256,uint256)`, which receives couple of parameters:
 
-- `uint256 cdp` - the `cdpId`
-- `int256 dink` - delta ink (collateral)
-- `int256 dart` - delta art (Dai).
+The function is called `frob(uint256, uint256, uint256)`, which receives the following parameters:
+
+- `uint256 cdp`: the `cdpId`
+- `int256 dink`: delta ink (collateral)
+- `int256 dart`: delta art (Dai).
```

Comment:
I'd suggest the following as alternative to the above.
```suggestion
- `int256 dink`: delta ink (collateral) `[wad]`
- `int256 dart`: delta art (Dai) `[wad]`
```

Reply:
I think it's better if we do both to add some context.